main/zoe/guide/contracts/loan.md

```diff
@@ -0,0 +1,143 @@
+* [`priceAuthority`](/zoe/guide/price-authority.md) - will be used for getting the current value of
+   collateral and setting liquidation triggers.
+* `autoswapInstance` - The running contract instance for an
+   [Autoswap](./autoswap.md) or [Multipool
+   Autoswap](./multipoolAutoswap.md) installation. The `publicFacet`
+   of the instance is used for producing an invitation to sell the
+   collateral on liquidation.
+* `periodAsyncIterable` - the [asyncIterable](https://javascript.info/async-iterators-generators) used for notifications
+   that a period has passed, on which compound interest will be
+   calculated using the `interestRate`.
+* `interestRate` - the rate in [basis points](https://www.investopedia.com/terms/b/basispoint.asp) that will be multiplied
+   with the debt on every period to compound interest.
+
+## IssuerKeywordRecord
+
+The following is used for all keyword records, regardless of role in
+the contract:
+
+* Keyword: 'Collateral' - The issuer/payment for the digital assets to be
+   escrowed as collateral.
+* Keyword: 'Loan' - The issuer/payment for the digital assets to be loaned
+   out.
+
+## The Lender
+
+The lender puts up the amount to be loaned to the borrower, but has no
+further actions. The loan is ongoing until it is paid back entirely or
+liquidated, at which point the lender receives a payout. This means
+that the payout for the lender will be in Loan-branded digital assets,
+not Collateral-brand. (The only exception to this is if the scheduling
+of liquidation triggers with the `priceAuthority` results in a error. In
+that case, we have no choice but to give the collateral to the lender.
+The borrower has already exited with their loan.)
+
+The lender will want the interest earned from the loan + their refund
+or the results of the liquidation. If the price of collateral drops
+before we get the chance to liquidate, the total payout could be zero.
+Therefore, the lender cannot `want` anything in their proposal.
+
+The lender must be able to exit on demand until borrowing occurs. If the exit rule was
+`waived`, a borrower would be able to hold on to their invitation and
+effectively lock up the lender's Loan forever.  When the borrowing
+occurs, the collateral is moved to a special collateral seat to
+prevent the lender from being able to exit with collateral before the
+contract ends through repayment or liquidation.
+
+## The Borrower
+
+The borrower receives an invitation to join, makes an offer with Zoe
+escrowing their collateral, and receives their loan. The borrower seat
+is exited at this point so the borrower gets the payout of their loan,
+but the borrower also gets an object (`borrowFacet`) as their `offerResult` that will
+let them continue to interact with the contract.
+
+Once the loan has started, the borrower can repay the loan in its
+entirety (at which the lender receives the loan amount back plus
+interest and the contract closes), or add more collateral to stave off
+liquidation.
```

Comment:
```suggestion
entirety at any time (at which point the lender receives the loan amount back plus
interest, and the contract closes), or add more collateral to prevent
liquidation.
```